runtime/src/bank/tests.rs

```diff
@@ -160,11 +160,7 @@ pub fn new_bank_from_parent_for_tests(
     slot: Slot,
 ) -> Arc<Bank> {
     let bank = Bank::new_from_parent(parent, collector_id, slot);
-    bank_forks
-        .write()
-        .unwrap()
-        .insert(bank)
-        .clone_without_scheduler()
+    bank_forks.write().unwrap().insert(bank)
```

Comment:
Insert returns `BankWithScheduler`, but the function returns `Arc<Bank>`, so I think we need the `clone_without_scheduler` here, don't we?

Reply:
v1.17 doesn't seem to have that.